<<<<<<< HEAD

int32 x_pos
int32 y_pos
int32 weight
int32 observed
=======
int64 x_pos
int64 y_pos
int64 weight
int64 observed
>>>>>>> 7da6d126a3fceaf755760ffbdcd978c598d9a4e4
